cargo-dist/src/tasks.rs

```diff
@@ -2882,6 +2985,8 @@ pub struct InstallReceipt {
     pub install_prefix: String,
     /// A list of all binaries installed by this app
     pub binaries: Vec<String>,
+    /// A list of all libraries installed by this app
+    pub libraries: Vec<String>,
```

Comment:
should probably also be called cdylibs..?

Reply:
I was wondering about how much we want to expose "cdylibs" vs "libraries" to the user, but I guess this isn't user-facing. We can be more specific about the technical details and separate other libraries into other categories here.